riscv/src/continuations/bootloader.rs

```diff
@@ -104,7 +105,7 @@ pub fn bootloader_preamble() -> String {
 ///   - The hash of the page *after* this chunk's execution
 ///   - For each level of the Merkle tree, except the root (1..=22):
 ///     - The hash (4 elements) of the sibling page
-pub fn bootloader(submachine_initialization: &[String]) -> String {
+pub fn bootloader_and_shutdown_routine(submachine_initialization: &[String]) -> String {
```

Comment:
I guess we should have another renaming PR, because everything is named "bootloader" now...